### ISSUE TITLE
[BUG] Cmd+Shift+Left selects to start of bullet instead of start of line

### ISSUE THREAD

**Issue by timmb** [vslinko/obsidian-outliner#376]
**Describe the bug**

When I'm on a multi-line bullet and I press CMD+SHIFT+Left (MacOS shortcut for go to start of line) it should select to the start of current line but instead selects to start of the whole bullet.

CMD+Left *does* work as expected: the caret moves to the start of the current line, and if pressed again it moves to the start of the bullet. So caret movement of CMD+Left and CMD+SHIFT+Left is inconsistent

**To Reproduce**
1. Create a bullet. Type until you are on the second line.
2. (On a Mac) Press CMD+Shift+Left. (I haven't tested on Windows but the equivlanet would be Shift+Home.)
3. Observe the caret moves to the *first* line of the bullet and selects the text between its original and new position.

**Expected behavior**
The caret should move to the start of the *current* line (and select the text between its original and new position).

**Desktop (please complete the following information):**
 - OS: MacOS 11.7
 - Obsidian Version: 1.0.0

**Additional context**
When I disable the Outline plugin, this bug disappears and the expected behaviour happens.

**Comment by dsteinbock** [comment]
I love this plugin and rely on it daily. This issue is my only complaint. 

Not sure how it works in other outliners but it's a text editor standard to select to the end of the current line with cmd-shift-left/right. 

There's already a plugin setting for cmd-A behavior to select the current list item, so there would be no loss of functionality if this were implemented. 

I discovered another bug/inconsistency with this. Perhaps it points to a solution. If any characters are already selected, then cmd-shift-left DOES work as expected.

Steps to reproduce:
1. Create a bullet. Type until you are on the second line.
2. (On a Mac) Press Shift-Left to select 1 character.
3. Press CMD+Shift+Left. 

The current line is selected, as expected.
